pkg/playbook/app/installer/capi.go

```diff
@@ -112,7 +112,7 @@ func (r CAPIInstaller) Install(pcc certificate.PEMCollection) error {
 	// Generate random password for temporary P12 bundle
 	bundlePassword := vcertutil.GeneratePassword()
 
-	content, err := packageAsPKCS12(pcc, bundlePassword)
+	content, err := packageAsPKCS12(pcc, bundlePassword, false)
```

Comment:
wouldn't this need to be `packageAsPKCS12(pcc, bundlePassword, r.UseLegacyP12)` as well?

Reply:
Yep, fixed